nav2_tasks/include/nav2_tasks/global_localization_service_client.hpp

```diff
@@ -15,24 +15,24 @@
 #ifndef NAV2_TASKS__GLOBAL_LOCALIZATION_SERVICE_CLIENT_HPP_
 #define NAV2_TASKS__GLOBAL_LOCALIZATION_SERVICE_CLIENT_HPP_
 
-#include "nav2_tasks/service_client.hpp"
+#include "nav2_util/service_client.hpp"
 #include "std_srvs/srv/empty.hpp"
 
 namespace nav2_tasks
 {
 
-class globalLocalizationServiceClient : public ServiceClient<std_srvs::srv::Empty>
+class globalLocalizationServiceClient : public nav2_util::ServiceClient<std_srvs::srv::Empty>
```

Comment:
Should global be uppercase?

Reply:
Yes. Fixed. As well as a bunch of other name violations in the nav2_utils package. I addressed all the functions in the main nav2_utils library, but left the libraries that came from AMCL and map server